src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -1130,8 +1130,8 @@ objects:
 
     | Source type | Description |
     |---|---|
-    | **Data element from the newest event for a program stage** | This source type works the same way as **Data element from the newest event in the current program**, except that it only evaluates values from one program stage.<br>     <br>This source type can be useful in program rules where the same data element is used in several program stages, and a rule needs to evaluate the newest data value from within one specific stage. |
-    | **Data element from the newest event in the current program** | This source type is used when a program rule variable needs to reflect the newest known value of a data element, regardless of what event the user currently has open.<br>     <br>This source type is populated slightly differently in **Tracker Capture** and **Event Capture** apps:<br>     <br>**Tracker Capture**: the program rule variable will be populated with the newest data value collected for the given data element within the enrollment.<br>     <br>**Event Capture**: the program rule variable will be populated with the current events data. <br>**NB** Future dates are "newer" than current or past dates. |
+    | **Data element from the newest event for a program stage** | This source type works the same way as **Data element from the newest event in the current program**, except that it only evaluates values from one program stage.<br>     <br>This source type can be useful in program rules where the same data element is used in several program stages, and a rule needs to evaluate the newest data value from within one specific stage. <br>     <br>In order to know what event is the newest, the report date (event date) is used. Take into account that if you have many events with the same report date, the system choose one randomly.|
```

Comment:
This may not be correct now. I think we have started using the time as well? @enricocolasante thoughts?

Reply:
Yes, we are not choosing randomly, we use the `createdAt` field for secondary order.

Reply:
@dayures If you can modify both the statements to reflect what Enrico said, we can take this PR forward. 

In order to know what event is the newest, the report date (event date) is used. If you have many events with the same report date, the system choose the one with the latest `createdAt` property of the event.

Reply:
@ameenhere @enricocolasante thanks for the update. I am going to fix the change. In which version the use of createdAt as second filter was introduced, in order to include it in the doc?

Reply:
The fix was introduced in 2.41.2, so it will be present in 2.42 and it is present in master

Reply:
@ameenhere change done.